consoleme/lib/aws.py

```diff
@@ -503,11 +503,11 @@ async def fetch_sqs_queue(account_id: str, region: str, resource_name: str) -> d
         result["TagSet"] = [{"Key": k, "Value": v} for k, v in tags.items()]
     if result.get("CreatedTimestamp"):
         result["created_time"] = datetime.utcfromtimestamp(
-            int(result["CreatedTimestamp"])
+            int(float(result["CreatedTimestamp"]))
```

Comment:
What's going on here?

Reply:
CreatedTimestamp could be an epoch timestamp with a decimal, IE "123456.123"

int("123456.123") results in an error, but int(float("123456.123")) does not

![image](https://user-images.githubusercontent.com/7538233/135360246-97cb1ce6-18c1-46ae-acc2-8ac1ad168834.png)

Reply:
Ah I see! Thanks